transforms/code/code2parquet/python/README.md

```diff
@@ -66,136 +69,60 @@ Each file contained within the ZIP is transformed into a distinct row within the
-
-To execute the script with S3 functionality, we utilize minio. 
-
-```make minio-start``` - To start minio server
-```make minio-load``` - To load the data into minio
-```make minio-stop``` - To stop the minio server
-
-Please consult the documentation for further guidance: [transform-s3-testing](../../data-processing-lib/doc/transform-s3-testing.md) 
-
-** Run script using make targets **
-```run-s3-sample``` : Starts minio server and load data from [test-data](test-data/input/) into local minio for S3 access and runs src/ code2parquet_s3.py
 
+## Running
 
-**Run the script without any command-line arguments.**
-
-```
-make venv
-source venv/bin/activate
-cd src
-python code2parquet_s3.py
-```
-
-**Run the script via command-line** 
-
-```
-python code2parquet.py \
-    --detect_programming_lang True \
-    --snapshot github \
-    --domain code \
-    --data_s3_cred '{"access_key": "localminioaccesskey", "secret_key": "localminiosecretkey", "url": "http://localhost:9000"}' \
-    --data_s3_config '{"input_folder": "test/code2parquet/input", "output_folder": "test/code2parquet/output"}' \
-    --data_files_to_use '[".zip"]'
-```
-
-The output directory will contain both the new genrated parquet files  and the `metadata.json` file.
-
-## Metadata Fields
-
-The metadata.json file contains following essential information regarding the processing of raw data files to Parquet format:
-
-`total_files_given`: Total number of raw data files provided for processing.
-`total_files_processed`: Number of files successfully processed and converted to Parquet format.
-`total_files_failed_to_processed`: Count of files that encountered processing errors and failed conversion.
-`total_no_of_rows`: Aggregate count of rows across all successfully processed files.
-`total_bytes_in_memory`: Total memory usage in bytes for all processed data files.
-`failure_details`: Information about files that failed processing, including their paths and error messages.
-
-## Building the Docker Image
-```
-% make image 
-```
-## Run using docker image
-
+### Launched Command Line Options
+When running the transform with the Ray launcher (i.e. TransformLauncher),
+the following command line arguments are available in addition to
+[the options provided by the launcher](../../../../data-processing-lib/doc/ray-launcher-options.md).
+
+```shell
+  --code2parquet_supported_langs_file CODE2PARQUET_SUPPORTED_LANGS_FILE
+                        Path to file containing the list of supported languages
+  --code2parquet_detect_programming_lang CODE2PARQUET_DETECT_PROGRAMMING_LANG
+                        generate programming lang
+ ```
+
+These correspond to the configuration keys described above.
+
+### Running the samples
+To run the samples, use the following `make` targets
```

Comment:
Please add make venv command before running the below commands.

Reply:
This is a good point, although when running `make run-cli-sample`, for example, it prints an error message indicating `make venv` needs to be run.